xpdacq/beamtime.py

```diff
@@ -329,7 +329,7 @@ def _scanplan_name_parser(self, sp_name):
         _Tramp_optional_params = ['det', 'subs_dict']
         _tseries_optional_params = ['det', 'subs_dict']
          
-        parsed_object = sp_name.split('_', maxsplit = 4)
+        parsed_object = sp_name.split('_', maxsplit = 4+3) # 3 is error tolerance
```

Comment:
yes, I didn't know what I was doing either....This has been corrected in current PR